sdk/python/lib/pulumi/runtime/settings.py

```diff
@@ -98,6 +95,73 @@ def __init__(
         else:
             self.engine = None
 
+    def __repr__(self):
+        return f"Settings: {self.engine}:{self.monitor}:{self.project}:{self.stack}"
```

Comment:
I'll likely remove this, but if we have a good idea of what we'd _actually_ want a `__repr__` to look like, I'd love to hear it.  Inspecting some of these objects is currently not very useful.